### PR TITLE
Ie11 bug fixes

### PR DESCRIPTION
**Pull request by scpaye506**

### CODE REVIEW

File: app/assets/stylesheets/redesign/_config.sass [hunk 1]
```diff
@@ -62,3 +62,8 @@ $maker-track: $teal-light
 $growth-track: $pink-light
 $spotlight-track: $mint-light
 
+
+// MIXINS
+@mixin font-family-mixin($family, $weight)
```
Comment (scpaye506) [inline app/assets/stylesheets/redesign/_config.sass]: @rflorence sorry, i had to add a mixin before you got yours in! 😛

Comment (rflorence) [reply app/assets/stylesheets/redesign/_config.sass]: Hey my PR was a few minutes earlier!

File: app/assets/stylesheets/redesign/_config.sass [hunk 2]
```diff
@@ -62,3 +62,8 @@ $maker-track: $teal-light
...
+  font-family: $family, sans-serif
+  font-weight: $weight
```
Comment (rflorence) [inline app/assets/stylesheets/redesign/_config.sass]: I haven't been seeing any difference in font-weight values with these Montserrat fonts. Have you?

File: app/assets/stylesheets/redesign/_config.sass (hunk 1 above)
Comment (scpaye506) [reply app/assets/stylesheets/redesign/_config.sass]: ha, i totally didn't even see it

File: app/assets/stylesheets/redesign/_config.sass (hunk 2 above)
Comment (scpaye506) [reply app/assets/stylesheets/redesign/_config.sass]: do you mean just define the weights in the `fonts.sass` file?

Comment (scpaye506) [reply app/assets/stylesheets/redesign/_config.sass]: nevermind, i see what you mean

Comment (scpaye506) [reply app/assets/stylesheets/redesign/_config.sass]: looks like the -Light and -Regular otf files were wrong, just updated them and removed the `weight` param from this mixin

**Pull request merged**